### DISCUSSION TITLE
SOFAR HYD3600 ES

### DISCUSSION THREAD

**Discussion by djdarlo** [StephanJoubert/home_assistant_solarman#264] [Q&A]
Will this integration work with the above? I'm trying to help my friend integrate his with HA. We'd like to be able to read the data and also set the times for charging batteries etc. is this possible?

**Comment by sanderbaks** [comment]
I have a lightly different unit (12KTL-G3) but mine is not working yet.

**Comment by sanderbaks** [comment]
Never got this to work. I used another plugin: "Ginlong Solis PV portal integration" from HACS.
First setup and account on the m.ginlong.com website. Then data come through
